nui-input/src/main/java/org/terasology/input/device/MouseDevice.java

```diff
@@ -31,10 +31,22 @@
     Queue<MouseAction> getInputQueue();
 
     /**
-     * @return The current position of the mouse in screen space
+     * @return The current position of the first mouse pointer in screen space
      */
     Vector2i getPosition();
 
+    /**
+     * @return The current position of a mouse pointer in screen space
+     */
+    default Vector2i getPosition(int pointer) {
+        if (pointer == 0) {
+            return getPosition();
+        } else {
+            // Multiple pointers not supported, so return a zero value.
```

Comment:
I chose to go with an [UnsupportedOperationException](https://docs.oracle.com/javase/8/docs/api/java/lang/UnsupportedOperationException.html) instead, since `NotSupportedException` is not present in any versions of Android.